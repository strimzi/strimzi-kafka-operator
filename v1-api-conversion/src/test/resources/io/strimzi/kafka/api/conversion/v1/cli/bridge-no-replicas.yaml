apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaBridge
metadata:
  name: my-bridge
spec:
  bootstrapServers: my-cluster-kafka-bootstrap:9092
  http:
    port: 8080