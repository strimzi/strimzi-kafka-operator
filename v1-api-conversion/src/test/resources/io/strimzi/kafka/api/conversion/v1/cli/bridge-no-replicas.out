---
apiVersion: kafka.strimzi.io/v1
kind: KafkaBridge
metadata:
  name: my-bridge
spec:
  bootstrapServers: my-cluster-kafka-bootstrap:9092
  http:
    port: 8080
  replicas: 1